dropwizard-logging/src/test/java/io/dropwizard/logging/FileAppenderFactoryTest.java

```diff
@@ -190,7 +190,10 @@ public void hasMaxFileSizeFixedWindow() throws Exception {
         RollingFileAppender<ILoggingEvent> appender = (RollingFileAppender<ILoggingEvent>) fileAppenderFactory.buildAppender(new LoggerContext());
 
         assertThat(appender.getRollingPolicy()).isInstanceOf(FixedWindowRollingPolicy.class);
+        assertThat(appender.getRollingPolicy().isStarted());
+        
         assertThat(appender.getTriggeringPolicy()).isInstanceOf(SizeBasedTriggeringPolicy.class);
+        assertThat(appender.getTriggeringPolicy().isStarted());
```

Comment:
I think there are some other issues[1] with the implementation in the else block. It seems like the `%d-%i` configuration should've never been supported. I believe there will need more intrusive changes in order to get a more robust rolling/triggering policy in those workflows.

[1] http://jira.qos.ch/browse/LOGBACK-1127?focusedCommentId=17110&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-17110
[2] http://jira.qos.ch/browse/LOGBACK-1143